### DISCUSSION TITLE
Linking my package to ARPACK-Armadillo

### DISCUSSION THREAD

**Discussion by algoquant** [opencollab/arpack-ng#390] [Q&A] [answered]
I'd like to call your ARPACK-Armadillo functions in my RcppArmadillo package.
Can you provide more information how I can do that?  I'm compiling my RcppArmadillo package in RStudio.
Where should I save the ARPACK-Armadillo files on my drive?
How should I modify my Makevars files?

**Comment by fghoussen** [comment, marked as answer]
This is `arpack-ng`, not `ARPACK-Armadillo` which seems to be avaliable here: https://github.com/yixuan/arpack-arma
